sdk/trace/tracetest/example_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package tracetest_test
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/otel/attribute"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/sdk/trace/tracetest"
+)
+
+func Example_simulateWorkflow() {
+	ctx := context.Background()
+
+	// Set up an in-memory span recorder and tracer provider.
+	recorder := tracetest.NewSpanRecorder()
+	tp := sdktrace.NewTracerProvider(
+		sdktrace.WithSpanProcessor(recorder),
+	)
+	defer tp.Shutdown(ctx)
+
+	tracer := tp.Tracer("example/workflow")
+
+	// Parent span "workflow"
+	ctx, workflowSpan := tracer.Start(ctx, "workflow")
+	workflowSpan.SetAttributes(attribute.String("workflow.phase", "start"))
```

Comment:
We can simplify things by removing those attributes, since they're not used in the example output.


```suggestion
```